Firestore/Example/Tests/SpecTests/FSTSyncEngineTestDriver.h

```diff
@@ -48,6 +50,11 @@ class Persistence;
 namespace core = firebase::firestore::core;
 namespace local = firebase::firestore::local;
 namespace model = firebase::firestore::model;
+namespace nanopb = firebase::firestore::nanopb;
+
+using ActiveTargetMap =
```

Comment:
Could you add comments to this declaration indicating what this should contain? Previously this was just a map of target id to query data, but now I have no idea what's going on here.

Reply:
Added comment.